e2e/fixtures/github-alert/rspress.config.ts

```diff
@@ -0,0 +1,19 @@
+import * as path from 'node:path';
+import { defineConfig } from 'rspress/config';
+
+export default defineConfig({
+  root: path.join(__dirname, 'doc'),
+  markdown: {
+    mdxRs: false,
+  },
+  themeConfig: {
```

Comment:
can delete this field

Reply:
i hope you mentioned themeConfig, i deleted that one